pl-backend/DTO/UserDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace pl_backend.DTO
+{
+    public class UserDto
+    {
+        public string FirstName { get; set; } = string.Empty;
```

Comment:
We are going to register with email probably
```suggestion
        public string Email{ get; set; } = string.Empty;
```